Program.fs

```diff
@@ -669,4 +588,82 @@ let rec main argv =
                     else
                         FemtoResult.ValidationFailed
 
-        int result
+
+type CLIArguments =
+    | [<MainCommand; ExactlyOnce>] Project of path : string
+    | Validate
+    | Resolve
+    | Preview
+
+    interface IArgParserTemplate with
+
+        member this.Usage =
+            match this with
+            | Project _ -> "specify the path to the F# project"
+            | Validate -> "check that the XML tags used in the F# project file is parsable and a npm package version can be calculated"
```

Comment:
Again not very important, but "is parsable" should be "are parsable" because "tags" is plural

Reply:
Good catch